src/paneer-tikka-masala.md

```diff
@@ -0,0 +1,46 @@
+# Paneer Tikka Masala
+
+![paneer tikka masala](pix/paneer-tikka-masala.webp "Paneer Tikka Masala served with Naan bread")
+
+## Ingredients
+
+- 250g cubes paneer
+- 1 onion
+- 1 capsicum
+- 200ml tomato puree 
+- 1 tsp red chilli powder
+- 1 tsp turmeric powder
+- 1 tsp coriander powder
+- 1/2 tsp garam masala powder
+- 100ml thick yogurt
+- 1 tsp fenugreek powder
+- dry fenugreek leaves
+- cardamom
+- cumin seeds
+- shaved ginger
+- 20ml garlic paste
+- 1 bowl corn
+
+## Directions
+
+- Take a glass container, and mix the cubed paneers with half a tea spoon of red chilli powder, turmeric powder, coriander powder, garam masala powder, fenugreek powder, and 50g yogurt.
+- Let the mix marinade for at least an hour.
+ 
+![paneer tikka masala](pix/paneer-tikka-masala-01.webp "Paneer ready to be marinated on the skewer")
+
+- Preheat the oven to 300°C, and place the marinated paneer cubes, with onion, capsicum, and tomato on the skewers. Let them grill in the oven using skewer support, by continously flipping them every 5 minutes until the outer crust looks crispy enough.
```

Comment:
```suggestion
- Preheat the oven to 300°C, and place the marinated paneer cubes, with onion, capsicum, and tomato on the skewers. Let them grill in the oven using skewer support, by continuously flipping them every 5 minutes until the outer crust looks crispy enough.
```

Reply:
Ignore this, looks like you fixed it while I was going through everything

Reply:
Yes, thanks for confirming.